src/yachtzee.ts

```diff
@@ -1042,28 +1021,46 @@ export function bestYachtzeeFamiliar(): Familiar {
     )
     .sort(
       (left, right) =>
-        leprechaunMeatBonus(
-          findLeprechaunMultiplier(right) *
-            (famWt +
-              (right.underwater ? (have($item`amulet coin`) ? 10 : 5) : famEquipWeightPenalty))
-        ) +
-        (right.underwater && have($item`amulet coin`) ? 50 : 0) -
-        leprechaunMeatBonus(
-          findLeprechaunMultiplier(left) *
-            (famWt +
-              (left.underwater ? (have($item`amulet coin`) ? 10 : 5) : famEquipWeightPenalty))
+        numericModifier(
+          right,
+          "meat drop",
+          famWt,
+          right.underwater || have($effect`Driving Waterproofly`) || have($effect`Wet Willied`)
```

Comment:
Do not do this logic in the middle of your numericModifier call, I would make this either be the output of a more destructured function or at the very least name a const